developer-docs-site/docs/concepts/staking.md

```diff
@@ -9,18 +9,14 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 
 ## Concept
 
-:::tip
+:::tip Consensus
 We strongly recommend that you read the consensus section of the [Life of a Transaction](/guides/basics-life-of-txn#consensus) before proceeding further. 
 :::
 
 In a distributed system like blockchain, executing a transaction is different from updating the state of the ledger and persisting the results in storage. An agreement, i.e., consensus, must be reached by a quorum of validators on the ordering of transactions and their execution results before the results are persisted in storage and the state of the ledger is updated. 
 
 A validator can participate in the consensus process. However, the validator can acquire the voting power only when they stake, i.e., place their utility coin into escrow. To encourage validators to participate in the consensus process, each validator's vote weight is made proportionate to the amount of validator's stake. In exchange, the validator is rewarded in proportion to the amount of validator's stake. Hence, the performance of the network, i.e., consensus, is aligned with the validator's interest, i.e., rewards.   
```

Comment:
I think we should also mention that there's currently no slashing implemented. Validators at most miss their rewards. Not sure if we should also mean that this might change in the future though